DTS-STN/secure-client-hub
# Update main from staging (July 26th, 2024)

### Changelog
fix:Updated footer ternary to use publicRuntimeConfig value
fix:Update contact us mapper to accommodate SIN link
fix:Update Dockerfile to include root cert

### Description of proposed changes:
This PR updates the main branch to be in line with the changes on staging as of July 26th, 2024. This includes the fix for the footer links, the change to the contact us mapper so the SIN link shows up, and the change to the Dockerfile to include the root cert so our MSCA-NG call works on prod.

### What to test for/How to test
1. Pull in branch
2. Type `npm run dev`
3. Ensure the footer links are redirecting where they should and the contact us page shows a SIN link
